Automattic/pocket-casts-android
# Display Ending story

## Description

This adds the Ending story UI.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_4049

I asked about better support for smaller devices here: p1729240847073459-slack-C041BGTFQ5R

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the Ending story and verify it.

## Screenshots or Screencast 

### Preview

| Regular | Small |
| - | - |
| ![Screenshot_20241018-104503](https://github.com/user-attachments/assets/4c3c2c5b-addf-44c9-a3c4-d1c20c2bdbc1) | ![smol](https://github.com/user-attachments/assets/e70f96ed-88a6-4e6a-a16e-f8ba22ad3db3) |

### Animation

https://github.com/user-attachments/assets/8d47864f-9a52-4f3b-abed-ef7a6cb1fab8

## Checklist
 
#### I have tested any UI changes...
